scipy/special/tests/test_mpmath.py

```diff
@@ -563,24 +563,44 @@ def test_cospi_zeros():
 # ------------------------------------------------------------------------------
 
 @check_version(mpmath, '0.19')
-def test_dn_quarter_period():
-    def dn(u, m):
-        return sc.ellipj(u, m)[2]
-
-    def mpmath_dn(u, m):
-        return float(mpmath.ellipfun("dn", u=u, m=m))
-
+def test_ellipj():
```

Comment:
I'd need to step through to confirm that this is working as I expect, but the intent looks good.